dashboard/config/routes.rb

```diff
@@ -712,8 +712,10 @@
 
     resources :zendesk_session, only: [:index]
 
-    post '/report_abuse', to: 'report_abuse#report_abuse'
-    get '/report_abuse', to: 'report_abuse#report_abuse_form'
+    controller :report_abuse do
+      get :report_abuse_form, path: '/report_abuse'
+      post :report_abuse, path: '/report_abuse'
```

Comment:
please move the `path` option onto the `controller` line. if this doesn't work, you can switch to using `resources only: []` instead of `controller`

Reply:
It looks like resourceful routes do not work in this instance since the controller is not creating different routes for each controller action.
There are no routes such as:
```
GET  /report_abuse/report_abuse_form
POST /report_abuse/report_abuse   
```
Instead, we have:
```
report_abuse#report_abuse_form
POST /report_abuse/report_abuse        → report_abuse#report_abuse
```
The only alternative to how the routes are currently defined is using a controller block and specifying the path.

Reply:
We could also update these method names to more RESTFUL names, such as `new` and `create`, but we would have to update any reference to the original path.

Reply:
good point -- these ones are hard to improve without changing the routes.